packages/falso/src/lib/code-snippet-data.ts

```diff
@@ -0,0 +1,146 @@
+import type { CodeSnippetLang } from './code-snippet';
+
+type CodeSnippetLangToCodeSnippetMap = { [key in CodeSnippetLang]: string };
+
+export const codeSnippetLangToCodeSnippet: CodeSnippetLangToCodeSnippetMap = {
```

Comment:
This isn't our pattern. Please use a JSON file. See [Sports](https://github.com/ngneat/falso/blob/main/packages/falso/src/lib/sports.json) for example.

Reply:
The issue #129 uses template literals for the example data which aren't supported by JSON. What would you recommend instead?

Reply:
use single quotes for strings or escape them.

Reply:
thanks, I've pushed another update